configure.ac

```diff
@@ -465,6 +465,12 @@ then
 	*-*-vxworks*)
 	    ac_sys_system=VxWorks
 	    ;;
+	*-*-emscripten)
+	    ac_sys_system=Emscripten
+	    ;;
+	*-*-wasi)
```

Comment:
Doesn't this prevent `wasm32-wasi` from working due to the due to the 2 `-`?

Reply:
At this point in the configure script ``wasm32-wasi`` is already canonicalized to ``wasm32-unknown-wasi``.